client/signup/steps/user-type/form.jsx

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
Hm, is this the right place to add this form? I don't think so. Let's keep it in `/jetpack/connect` for now - this is not a WP.com signup step, so it shouldn't be here.